resources/types/Genome.yaml

```diff
@@ -0,0 +1,73 @@
+# This specification describes how a Workspace Genome object is transformed into a document that is
+# indexed by the KBaseSeachEngine.
+
+# The name of the type in the search system.
+global-object-type: Genome
+# The data source of the objects, in this case the Workspace.
+storage-type: WS
+# The name of the type in the data source.
+storage-object-type: KBaseGenomes.Genome
+
+# The following rules describe which fields are extracted from the source object and transforms
+# performed upon said fields.
+indexing-rules:
+    -
+        # Extract a string from the /id path. The key name will be id.
+        path: id
+        keyword-type: string
+    -
+        # Extract a string from the /domain path. The key name will be domain.
+        path: domain
+        keyword-type: string
+    -
+        # Extract a full-text search field from the /taxonomy path. A full-text field is parsed
+        # into tokens which can be searched on individually, as opposed to a string field which is
+        # an opaque block and must be matched in its entirety.
+        # The key name will be taxonomy.
+        path: taxonomy
+        full-text: true
+    -
+        # Extract a full-text search field from the /scientific_name path.
+        # The key name will be scientific_name.
+        path: scientific_name
+        full-text: true
+    -
+        # Extract the number of features from a list at the /features path. The special {size}
+        # element directs the parser to return the size of list or mapping. Since this number
+        # will be an integer, the keyword type is set a such.
```

Comment:
of list -> of a list
set a such -> set as such
Fixed this locally, it'll show up in a later PR.